README.md

```diff
@@ -55,7 +55,8 @@ ktlint {
     debug = true
     verbose = true
     android = false
-    reporter = "checkstyle"
+    outputToConsole = true
+    reporters = ["plain", "checkstyle"]
```

Comment:
This actually isn't valid kotlin. I don't know if groovy will do this coercion either. You may need the actual type `ReporterType.PLAIN` here.

Reply:
@JLLeitschuh Any idea, how to do it wit `setOf()` ?

Reply:
I don't understand the question

Reply:
To be honest, I don't know, how to write the syntax for array/set in gradle file for `reporters`. Is there any way, how to do it? To pass multiple reporters? @JLLeitschuh